src/main/java/CosLayer.mdl

```diff
@@ -127,7 +129,12 @@ type CosStream extends CosDict {
 
 % File descriptor dictionary		
 type CosFileDescriptor extends CosDict {
-	
+
+}
+
+type CosFileSpecification extends CosDictionary {
```

Comment:
Missing Entity error here, I presume that the xtextmodel-syntax project is out of date?
```
[ERROR] ERROR:Couldn't resolve reference to Entity 'CosDictionary'. (file:/home/cfw/Dropbox/Desktop/veraPDF/dev/veraPDF-model/src/main/java/CosLayer.mdl line : 135)
```